server.js

```diff
@@ -1088,6 +1088,32 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Clojars version integration
+camp.route(/^\/clojars\/v\/(.+)((\/)(.+))?\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var clojar = match[1];  // eg, `prismic` or `foo/bar`.
+  var format = match[2];
```

Comment:
The format would then be `match[5]`, but that's only because of a number of useless bracket matchings in the regex. `/^\/clojars\/v\/(.+)\.(svg|png|gif|jpg|json)$/` is likely to work a lot better (and would keep `match[2]`).